### PR TITLE
Update method.xml

### PR DESCRIPTION
**Pull request by vedamanavi**
adding support for latn_qwerty_tly.xml

### CODE REVIEW

File: res/xml/method.xml [hunk 1]
```diff
@@ -34,6 +34,7 @@
...
   <subtype android:label="%s" android:languageTag="sv" android:imeSubtypeLocale="sv_SE" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="script=latin,default_layout=latn_qwerty_se,extra_keys=accent_aigu:á|accent_trema:ä:ö|accent_ring:å|€"/>
   <subtype android:label="%s" android:languageTag="tr" android:imeSubtypeLocale="tr_TR" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="script=latin,default_layout=latn_qwerty_tr,extra_keys=accent_cedille:ç:ş|accent_trema:ö:ü|accent_circonflexe:â:î:û|₺|ı|ğ"/>
+  <subtype android:label="%s" android:languageTag="tly" android:imeSubtypeLocale="tly_AZ" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="script=latin,default_layout=latn_qwerty_tly.xml,extra_keys=accent_aigu:á:ú|accent_circonflexe:â:ê:ı|accent_caron:š:ž"/>
```
Comment (Julow) [inline res/xml/method.xml:37]:
```suggestion
  <subtype android:label="%s" android:languageTag="tly" android:imeSubtypeLocale="tly_AZ" android:imeSubtypeMode="keyboard" android:isAsciiCapable="true" android:imeSubtypeExtraValue="script=latin,default_layout=latn_qwerty_tly,extra_keys=accent_aigu:á:ú|accent_circonflexe:â:ê:ı|accent_caron:š:ž"/>
```

Comment (Julow) [inline res/xml/method.xml:37]: The `accent_circonflexe:â:ê:ı` part means to not show the circumflex accent if all of `â`, `ê` and `ı` are already on the keyboard but `ı` isn't related to it.
Do you want to add a new combination (for example `accent_circonflexe + i = ı`) in [KeyModifier.java](https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/juloo.keyboard2/KeyModifier.java#L563) ?
Or do you mean to add `ı` as an extra key ? (by changing the `:` before it into `|`)

**Review by Julow — COMMENTED** [review]
Thanks :) Sorry for the rush, I now plan to release in 7 days instead of tomorrow.

File: res/xml/method.xml (hunk 1 above)
Comment (Julow) [reply res/xml/method.xml]: The `extra_keys` part defines what keys are visible when the user uses a different layout. For example, for users that have several languages installed.

**Comment by vedamanavi** [comment]
@Julow Could you please check it now? Thanks.

**Review by Julow — APPROVED** [review]
Looks good, thanks!

**Pull request merged**